tests/test_api.py

```diff
@@ -165,6 +166,25 @@ def test_run_only(package):
     assert "Surface Temperature" in results.df.variable.unique()
 
 
+def test_run_rewritten_scen_file(package, temp_dir):
+    # this used to fail with early implementations of MAGICC7
```

Comment:
So it doesn't anymore? Do we need to be able to test early implementations of MAGICC7?

Reply:
Not in this way (where we re-write a SCEN file as a SCEN7 file and then run)

Reply:
Still don't get it ... :-/
This is testing SCEN to SCEN7?
Early versions of MAGICC7 could not read a re-written SCEN file?

Reply:
Yep so it reads in a SCEN file, and re-writes it straight to a SCEN7 file. Early versions of MAGICC7 couldn't handle this as the region set which is in the RCPs {R5OECD, R5REF, R5MAF, R5LAM, R5ASIA, WORLD, BUNKERS} didn't exist in early versions of MAGICC7 and had been deprecated to {R6OECD90, R6REF, R6MAF, R6LAM, R6ASIA, WORLD, BUNKERS}, hence the need for the 'emergency renaming'. That's fixed now and so the emergency renaming is no longer required.

I don't want to put this in the docs as I think it's better to just be happy it's fixed rather than confuse people by telling about something which used to be broken.

Reply:
Cool! Then can we just remove the comment (as it confused me :grin:)?

Reply:
smart